Doc/Makefile

```diff
@@ -107,7 +107,7 @@ clean:
 
 venv:
 	$(PYTHON) -m venv venv
-	./venv/bin/python3 -m pip install -U Sphinx
+	./venv/bin/python3 -m pip install sphinx~=1.5.6
```

Comment:
I think it might be better to specify the version in `.travis.yml` and leave the Makefile alone. My thinking is that locking Travis for testing makes sure it stays green and it's obvious what we are requiring for CI to pass, but leaving the Makefile alone makes it easier to update the docs to the latest version of Sphinx offline.

Python 2.7 also doesn't have the `venv` command in the Makefile but it does have the explicit pip call in `.travis.yml`.

Or maybe I'm just nuts and that logic is flawed. 😄